.gitmodules

```
@@ -0,0 +1,3 @@
+[submodule "stm32_i2s"]
+	path = stm32_i2s
+	url = https://github.com/YruamaLairba/stm32_i2s
```

Please, don't use submodules.
Cargo has powerful dependency control system.
```suggestion
[dependencies.stm32_i2s]
git = "https://github.com/YruamaLairba/stm32_i2s"
```
if you need to rename package (for example if you need to use 2 similar crate) add
```toml
package = "nename"
```

---

Sorry, the reason i use submodule is to avoid to have to do a push each time i fix something in smt32_i2s while ensuring other people can build to check if it work.